buildbot/configure.py

```diff
@@ -75,6 +75,11 @@ def do_configure(args):
         sycl_build_pi_hip_platform = args.hip_platform
         sycl_enabled_plugins.append("hip")
 
+    # all backends don't require 3rd party dependencies, so can be built/tested
```

Comment:
It looks like you intention is to enable build of all supported GPU targets in LLVM, not SYCL back-ends. Am I right?
If so, I suggest renaming the switch to `--enable-all-llvm-targets` to avoid confusion. Enabling SYCL back-ends requires building libclc and runtime library plug-ins in addition to LLVM targets.